beanstalkc.py

```diff
@@ -31,6 +31,15 @@
 DEFAULT_TTR = 120
 
 
+PY3 = sys.version_info[0] > 2
+if PY3:
+    b = lambda x: isinstance(x, bytes) and x or bytes(x, 'us-ascii')
+    s = lambda x: x.decode('us-ascii')
```

Comment:
Any reason not to do the following?

    b = lambda x: x if isinstance(x, bytes) else x.encode()
    s = lambda x: x.decode()

This would change the encoding from `us-ascii` to `utf-8`, but I am not sure why you chose `us-ascii` in the first place.